### PR TITLE
Fix `cargo spellcheck` failure in CI

### PR DESCRIPTION
**Pull request by cmichi**
See https://gitlab.parity.io/parity/ink/-/jobs/1010416.

### CODE REVIEW

File: .config/cargo_spellcheck.dic [hunk 1]
```diff
@@ -1,4 +1,4 @@
-80
+90
```
Comment (HCastano) [inline .config/cargo_spellcheck.dic:1]: Why's this changing to `90`?

File: .config/cargo_spellcheck.dic [hunk 2]
```diff
@@ -65,11 +65,14 @@ prepend
...
 Django/S
 IP/S
+ink!/S
 NFT/S
+SCALE/S
 accessor/S
 allocator/S
 backend/S
 blockchain/S
+callable/S
```
Comment (HCastano) [inline .config/cargo_spellcheck.dic:75]: Wouldn't the `/S` match `callables`/`callable's`? I don't think these are words...

**Review by HCastano — APPROVED** [review]

File: .config/cargo_spellcheck.dic (hunk 2 above)
Comment (cmichi) [reply .config/cargo_spellcheck.dic]: I switched to `/MS`, for matching only `callable's`.

Yeah I agree that it's not an ideal word, we use it quite often in the codebase though. Do you have a better suggestion?

File: .config/cargo_spellcheck.dic (hunk 1 above)
Comment (HCastano) [reply .config/cargo_spellcheck.dic:1]: Oh lol, I had no clue that's what the number was for

**Pull request merged**